projects/event-lambdas/src/event-api-lambda/__tests__/api-lambda.spec.ts

```diff
@@ -26,7 +26,8 @@ describe("Event API lambda", () => {
   beforeAll(async () => {
     try {
       await s3.listObjects({ Bucket: telemetryBucketName }).promise();
-    } catch (e) {
+    } catch (err) {
+      const e = err as Error
```

Comment:
Out of interest is this something that the newer version of `ts-jest` took issue with?